src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ProjectLockFileWatcher.cs

```diff
@@ -159,18 +159,21 @@ public int FilesChanged(uint cChanges, string[] rgpszFile, uint[] rggrfChange)
             // the kind of change since we are interested in all changes.
             _projectServices.ThreadingService.Fork(async () =>
             {
-                using (var access = await _projectLockService.WriteLockAsync())
+                await _projectServices.Project.Services.ProjectAsynchronousTasks.LoadedProjectAsync(async () =>
```

Comment:
Sounds like we should just catch that here and ignore - there's nothing to be done if the project's already unloaded.